Packages/com.unity.render-pipelines.core/Runtime/Utilities/CoreUtils.cs

```diff
@@ -890,7 +890,7 @@ static string GetTextureAutoName(int width, int height, string format, TextureDi
                 temp = string.Format("{0}x{1}{2}_{3}", width, height, mips ? "_Mips" : "", format);
             else
                 temp = string.Format("{0}x{1}x{2}{3}_{4}", width, height, depth, mips ? "_Mips" : "", format);
-            temp = String.Format("{0}_{1}_{2}", name == "" ? "Texture" : name, (dim == TextureDimension.None) ? "" : dim.ToString(), temp);
+            temp = String.Format("{0}_{1}_{2}", name?.Length == 0 ? "Texture" : name, (dim == TextureDimension.None) ? "" : dim.ToString(), temp);
```

Comment:
At this point, to add the null check, it is even better to rely on String.IsNullOrEmpty

```suggestion
            temp = String.Format("{0}_{1}_{2}", String.IsNullOrEmpty(name) ? "Texture" : name, (dim == TextureDimension.None) ? "" : dim.ToString(), temp);
```

Reply:
I benchmarked it and isNullOrEmpty actually does a string comparison instead of a null check, so is slower than this!
It is more readable however, so just say what you prefer. I can share the benchmark numbers tomorrow, but it was more than I expected

Reply:
Please do, I am curious about the difference.